src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -1366,6 +1370,14 @@ static void updateSfDriverParamValues(Map<String, Object> parameters, SFBaseSess
         if (session != null) {
           session.setClientResultChunkSize((int) entry.getValue());
         }
+      } else if (GEOGRAPHY_OUTPUT_FORMAT.equalsIgnoreCase(entry.getKey())) {
```

Comment:
Why don't we rename `snowparkParameters` (and the getter/setter methods accordingly) to something generic, like `otherParameters`? That way, we can just let the `else`-condition (i.e., if the parameter tag is not recognized as the belonging to the core JDBC set) add that parameter, with its key, to the `otherParameters` map. The Snowpark client should maintain a hardcoded list of all of the parameters it needs to use, but not JDBC. This way, we can evolve Snowpark and add new Snowpark parameters without having to modify JDBC as well.

Reply:
Sounds reasonable to me. I'm not sure why currently JDBC restrict parameters to a set of hard coded values, so I kept the original pattern here.
@sfc-gh-mknister does that sound good to you as well?

Reply:
Yes, this sounds good to me as well!